src/Components/Admin/common/errorModal.js

```diff
@@ -43,10 +47,23 @@ export default function ErrorModal({ error, resetErrorBoundary }) {
             <DialogContent>{message}</DialogContent>
             <DialogActions className={classes.buttonContainer}>
                 <Button
-                    onClick={resetErrorBoundary}
+                    onClick={() => {
+                        // TODO: send email to us
+                    }}
+                    className={classes.deleteButton}
+                >
+                    Report
+                </Button>
+                <Button
+                    onClick={() => {
```

Comment:
If I understand this code correctly, won't this log you out on _any_ error? So if you have a regex error and we show this error modal, if they hit close, this will log them out rather than just closing the modal.